raiden/messages.py

```diff
@@ -960,6 +964,25 @@ def from_event(cls, event):
             signature=EMPTY_SIGNATURE,
         )
 
+    @property
+    def message_hash(self) -> bytes:
+        return sha3(
+            pack_data(
+                (self.cmdid.value, "uint8"),
+                (b"\x00" * 3, "bytes"),  # padding
```

Comment:
Note: This doesn't need to be here.

The `message_hash` exists as a way to normalize the differences between the messages LockedTransfer, RemoveExpiredLock, UnlockTransfer. Roughly, the first has a lock, the second has a lockhash, and the third has a secret. This data must be validated, but it's not part of the balance proof in the smart contract, so we use the `additional_hash` as a way of allowing the off-chain client to check the integrity of the message.

The padding was here only because the code worked as it was, not because it was necessary. The `message_hash` only really needs the fields that are **not** in the balance proof.